packages/paste-core/utilities/media-object/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import renderer from 'react-test-renderer';
+import {Theme} from '@twilio-paste/theme';
+import {MediaObject, MediaFigure, MediaBody} from '../src';
+
+describe('MediaObject', () => {
+  it('should render', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <MediaObject>default</MediaObject>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render with center aligned children', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <MediaObject verticalAlign="center">center align</MediaObject>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render as another HTML element', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <MediaObject as="section">background single</MediaObject>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
I kind of think an enzyme test would be better than a snapshot test for a bunch of these: https://airbnb.io/enzyme/docs/api/ShallowWrapper/name.html#examples